source/slang/slang-ast-type.cpp

```diff
@@ -833,14 +852,21 @@ Val* ExtractExistentialType::_substituteImplOverride(
 
     (*ioDiff)++;
 
-    ExtractExistentialType* substValue = astBuilder->getOrCreate<ExtractExistentialType>(
-        substDeclRef,
-        as<Type>(substOriginalInterfaceType),
-        substOriginalInterfaceDeclRef);
+    InterfaceWithContext* substValue = nullptr;
```

Comment:
The name `InterfaceWithContext` is completely opaque to me. Can you help me understand the motivation behind this name?